client/state/rewind/test/selectors.js

```diff
@@ -385,3 +387,54 @@ describe( 'Backup staging site info', () => {
 		} );
 	} );
 } );
+
+jest.mock( 'calypso/state/selectors/get-rewind-backups' );
+
+describe( 'getFinishedBackupForSiteById()', () => {
+	const TEST_SITE_ID = 123;
+	const state = {};
+
+	beforeEach( () => {
+		getRewindBackups.mockReset();
+	} );
+
+	test( 'Should receive the finished backup.', () => {
+		const BACKUP_ID = 1;
```

Comment:
Can the const BACKUP_ID = 1 each test uses be pulled to the top with TEST_SITE_ID

Reply:
Right! Moving it (also creating a `TEST_BACKUP_ID_2`)